nix/modules/lanzaboote.nix

```diff
@@ -104,6 +104,7 @@ in
         ''}
 
         ${cfg.package}/bin/lzbt install \
+          --target-system ${config.boot.kernelPackages.stdenv.hostPlatform.system} \
```

Comment:
```suggestion
          --target-system ${config.nixpkgs.hostPlatform.system} \
```

Reply:
No, then, it should be `config.nixpkgs.targetPlatform.system` I believe.

Reply:
But this can create surprising behaviors IMHO.